nilearn/decoding/tests/test_searchlight.py

```diff
@@ -5,14 +5,14 @@
 from nibabel import Nifti1Image
 from sklearn.model_selection import KFold
 
+from nilearn.conftest import _rng
 from nilearn.decoding import searchlight
 
 
 def _make_searchlight_test_data(frames):
```

Comment:
Why not make rng an argument of the test in that case ?

Reply:
AFAICT because `rng` is a fixture and fixtures can only be passed as arguments to a test function (or another fixture) but not to a utility function (pretty sure it is more complicated than that but this is the super short version)